[2025-02-13 22:03:34,428: INFO: main: logging is implemented]
[2025-02-13 22:03:52,211: INFO: main: logging is implemented]
[2025-02-14 16:17:23,454: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:17:23,458: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:20:22,607: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:20:22,614: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:20:22,615: INFO: common: created directory at: artifacts]
[2025-02-14 16:20:22,616: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:21:53,434: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:21:53,434: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:21:53,437: INFO: common: created directory at: artifacts]
[2025-02-14 16:21:53,438: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:32:33,539: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:32:33,539: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:32:33,542: INFO: common: created directory at: artifacts]
[2025-02-14 16:32:33,543: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:33:41,604: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:33:41,604: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:33:41,604: INFO: common: created directory at: artifacts]
[2025-02-14 16:33:41,604: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:34:41,880: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:34:41,880: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:34:41,886: INFO: common: created directory at: artifacts]
[2025-02-14 16:34:41,887: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:35:27,050: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:35:27,051: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:35:27,052: INFO: common: created directory at: artifacts]
[2025-02-14 16:35:27,052: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:36:22,348: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:36:22,349: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:36:22,350: INFO: common: created directory at: artifacts]
[2025-02-14 16:36:22,351: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:36:41,871: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:36:41,871: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:36:41,874: INFO: common: created directory at: artifacts]
[2025-02-14 16:36:41,876: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:36:55,475: INFO: 3347790719: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: A60D:216E99:3C03A5:40A7CD:67AF38DF
Accept-Ranges: bytes
Date: Fri, 14 Feb 2025 12:36:55 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990021-FJR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1739536609.464480,VS0,VE6406
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: aebcba35cb118a3cc4d6e9c1af662d02ac304839
Expires: Fri, 14 Feb 2025 12:41:55 GMT
Source-Age: 6

]
[2025-02-14 16:39:41,050: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:39:41,052: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:39:41,053: INFO: common: created directory at: artifacts]
[2025-02-14 16:39:41,054: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:39:46,632: INFO: 1916786552: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: A60D:216E99:3C03A5:40A7CD:67AF38DF
Accept-Ranges: bytes
Date: Fri, 14 Feb 2025 12:39:47 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990025-FJR
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1739536787.045728,VS0,VE0
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: d8dc3baa78058ef81266c8e57e0d615735987f6f
Expires: Fri, 14 Feb 2025 12:44:47 GMT
Source-Age: 177

]
[2025-02-14 18:12:41,512: INFO: main: logging is implemented]
[2025-02-14 18:12:41,512: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-14 18:12:41,528: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 18:12:41,537: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 18:12:41,538: INFO: common: created directory at: artifacts]
[2025-02-14 18:12:41,539: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 18:12:42,942: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7D5F:15F9A4:10662E:25E172:67AF4F5A
Accept-Ranges: bytes
Date: Fri, 14 Feb 2025 14:12:43 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990029-FJR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1739542363.796677,VS0,VE624
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: edd9e9e9cc267c5e9fc3ac71b303837f3dded553
Expires: Fri, 14 Feb 2025 14:17:43 GMT
Source-Age: 0

]
[2025-02-14 18:12:43,049: INFO: main: stage {STAGE_NAME} completed]
[2025-02-14 19:21:04,860: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-14 19:54:58,111: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 19:54:58,133: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 19:54:58,135: INFO: common: created directory at: artifacts]
[2025-02-14 19:54:58,138: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-14 19:56:00,408: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 19:56:00,410: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 19:56:00,411: INFO: common: created directory at: artifacts]
[2025-02-14 19:56:00,412: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 13:03:12,171: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 13:03:13,101: INFO: main: logging is implemented]
[2025-02-15 13:03:13,101: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:03:13,112: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:03:13,125: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:03:13,127: INFO: common: created directory at: artifacts]
[2025-02-15 13:03:13,127: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 13:03:13,127: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 13:03:13,240: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 13:03:13,240: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:03:13,241: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:03:13,241: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:03:13,241: INFO: common: created directory at: artifacts]
[2025-02-15 13:03:13,241: ERROR: main: 'function' object has no attribute 'tokenizer_name']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 35, in <module>
    data_processing_pipeline.initiate_data_transformation()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_processing_pipeline.py", line 16, in initiate_data_transformation
    data_ingestion = DataTransformation(config=data_transformation_config)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_transformation.py", line 18, in __init__
    self.tokenizer = AutoTokenizer.from_pretrained(config.tokenizer_name)
AttributeError: 'function' object has no attribute 'tokenizer_name'
[2025-02-15 13:04:01,103: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 13:04:01,246: INFO: main: logging is implemented]
[2025-02-15 13:04:01,246: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:04:01,246: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:04:01,246: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:04:01,246: INFO: common: created directory at: artifacts]
[2025-02-15 13:04:01,246: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 13:04:01,246: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 13:04:01,355: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 13:04:01,355: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:04:01,355: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:04:01,355: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:04:01,355: INFO: common: created directory at: artifacts]
[2025-02-15 13:04:01,355: ERROR: main: 'function' object has no attribute 'tokenizer_name']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 35, in <module>
    data_processing_pipeline.initiate_data_transformation()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_processing_pipeline.py", line 16, in initiate_data_transformation
    data_ingestion = DataTransformation(config=data_transformation_config)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_transformation.py", line 18, in __init__
    self.tokenizer = AutoTokenizer.from_pretrained(self.config.tokenizer_name)
AttributeError: 'function' object has no attribute 'tokenizer_name'
[2025-02-15 13:06:51,544: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 13:06:51,704: INFO: main: logging is implemented]
[2025-02-15 13:06:51,704: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:06:51,704: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:06:51,704: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:06:51,704: INFO: common: created directory at: artifacts]
[2025-02-15 13:06:51,704: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 13:06:51,704: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 13:06:51,802: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 13:06:51,802: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:06:51,802: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:06:51,802: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:06:51,802: INFO: common: created directory at: artifacts]
[2025-02-15 13:06:51,802: ERROR: main: 'function' object has no attribute 'tokenizer_name']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 35, in <module>
    data_processing_pipeline.initiate_data_transformation()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_processing_pipeline.py", line 16, in initiate_data_transformation
    data_ingestion = DataTransformation(config=data_transformation_config)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_transformation.py", line 18, in __init__
    self.tokenizer = AutoTokenizer.from_pretrained(self.config.tokenizer_name)
AttributeError: 'function' object has no attribute 'tokenizer_name'
[2025-02-15 13:09:34,053: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 13:09:34,211: INFO: main: logging is implemented]
[2025-02-15 13:09:34,211: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:09:34,211: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:09:34,211: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:09:34,211: INFO: common: created directory at: artifacts]
[2025-02-15 13:09:34,211: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 13:09:34,211: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 13:09:34,305: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 13:09:34,305: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:09:34,305: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:09:34,305: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:09:34,305: INFO: common: created directory at: artifacts]
[2025-02-15 13:09:34,305: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 13:09:37,105: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 14:51:49,687: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 16:44:05,594: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 16:44:05,602: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 16:44:05,603: INFO: common: created directory at: artifacts]
[2025-02-15 16:44:05,604: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 16:46:04,140: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 16:46:04,142: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 16:46:04,144: INFO: common: created directory at: artifacts]
[2025-02-15 16:46:04,145: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:20:58,539: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:20:58,541: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:20:58,542: INFO: common: created directory at: artifacts]
[2025-02-15 17:20:58,543: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:22:07,113: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:22:07,117: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:22:07,118: INFO: common: created directory at: artifacts]
[2025-02-15 17:22:07,120: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:23:06,950: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:23:06,952: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:23:06,953: INFO: common: created directory at: artifacts]
[2025-02-15 17:23:06,954: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:30:07,674: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:30:07,986: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:30:07,986: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:30:07,986: INFO: common: created directory at: artifacts]
[2025-02-15 17:30:07,986: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:31:38,234: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:31:38,436: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:31:38,438: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:31:38,439: INFO: common: created directory at: artifacts]
[2025-02-15 17:31:38,440: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:32:43,889: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:32:44,096: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:32:44,101: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:32:44,102: INFO: common: created directory at: artifacts]
[2025-02-15 17:32:44,102: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:36:29,077: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:36:29,274: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:36:29,275: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:36:29,276: INFO: common: created directory at: artifacts]
[2025-02-15 17:36:29,276: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:42:25,525: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:42:25,525: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:42:25,528: INFO: common: created directory at: artifacts]
[2025-02-15 17:42:25,529: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:44:21,769: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:44:21,771: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:44:21,773: INFO: common: created directory at: artifacts]
[2025-02-15 17:44:21,773: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:49:20,306: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:49:20,577: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:49:20,584: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:49:20,584: INFO: common: created directory at: artifacts]
[2025-02-15 17:49:20,584: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:54:36,200: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:54:36,207: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:54:36,209: INFO: common: created directory at: artifacts]
[2025-02-15 17:54:36,209: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 18:00:39,931: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 18:00:39,932: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 18:00:39,934: INFO: common: created directory at: artifacts]
[2025-02-15 18:00:39,934: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 19:11:59,886: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 19:12:00,329: INFO: main: logging is implemented]
[2025-02-15 19:12:00,331: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:12:00,331: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:12:00,335: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:12:00,335: INFO: common: created directory at: artifacts]
[2025-02-15 19:12:00,337: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 19:12:00,337: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 19:12:00,406: ERROR: main: [Errno 22] Invalid argument: 'artifacts\\data_ingestion\\samsum_dataset\\test\\data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 18, in <module>
    data_ingestion_pipeline.initiate_data_ingestion()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_ingestion_pipeline.py", line 18, in initiate_data_ingestion
    data_ingestion.extract_zip_file()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_ingestion.py", line 42, in extract_zip_file
    zip_ref.extractall(unzip_path)
  File "C:\Users\amitt\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1641, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\amitt\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1695, in _extract_member
    open(targetpath, "wb") as target:
OSError: [Errno 22] Invalid argument: 'artifacts\\data_ingestion\\samsum_dataset\\test\\data-00000-of-00001.arrow'
[2025-02-15 19:22:12,694: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 19:22:12,993: INFO: main: logging is implemented]
[2025-02-15 19:22:12,993: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:22:12,993: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:22:12,993: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:22:12,993: INFO: common: created directory at: artifacts]
[2025-02-15 19:22:12,993: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 19:22:13,009: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 19:22:13,058: ERROR: main: [Errno 22] Invalid argument: 'artifacts\\data_ingestion\\samsum_dataset\\test\\data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 18, in <module>
    data_ingestion_pipeline.initiate_data_ingestion()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_ingestion_pipeline.py", line 18, in initiate_data_ingestion
    data_ingestion.extract_zip_file()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_ingestion.py", line 42, in extract_zip_file
    zip_ref.extractall(unzip_path)
  File "C:\Users\amitt\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1641, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\amitt\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1695, in _extract_member
    open(targetpath, "wb") as target:
OSError: [Errno 22] Invalid argument: 'artifacts\\data_ingestion\\samsum_dataset\\test\\data-00000-of-00001.arrow'
[2025-02-15 19:23:11,260: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 19:23:11,541: INFO: main: logging is implemented]
[2025-02-15 19:23:11,541: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:23:11,541: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:23:11,541: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:23:11,541: INFO: common: created directory at: artifacts]
[2025-02-15 19:23:11,541: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 19:23:14,261: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E443:3403CA:793D23:80E91F:67B0B160
Accept-Ranges: bytes
Date: Sat, 15 Feb 2025 15:23:14 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990023-FJR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1739632992.357555,VS0,VE1704
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 504ab52a68fee4b87c7e908a78acdf170fb27fac
Expires: Sat, 15 Feb 2025 15:28:14 GMT
Source-Age: 1

]
[2025-02-15 19:23:14,367: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 19:23:14,367: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:23:14,367: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:23:14,367: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:23:14,367: INFO: common: created directory at: artifacts]
[2025-02-15 19:23:14,367: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 19:23:17,296: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 19:23:17,296: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:23:17,297: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 19:25:21,643: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 19:25:21,935: INFO: main: logging is implemented]
[2025-02-15 19:25:21,935: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 19:25:21,935: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:25:21,935: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:25:21,935: INFO: common: created directory at: artifacts]
[2025-02-15 19:25:21,935: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 19:25:22,311: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E443:3403CA:793D23:80E91F:67B0B160
Accept-Ranges: bytes
Date: Sat, 15 Feb 2025 15:25:22 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990033-FJR
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1739633122.289859,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 6a3a3544e17f50febdacc4941ea0082a12331cf3
Expires: Sat, 15 Feb 2025 15:30:22 GMT
Source-Age: 129

]
[2025-02-15 19:25:22,405: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 19:25:22,405: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:25:22,405: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:25:22,405: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:25:22,405: INFO: common: created directory at: artifacts]
[2025-02-15 19:25:22,421: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 19:25:25,014: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 19:25:25,015: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 19:25:25,016: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 20:34:05,027: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 21:00:31,200: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:00:31,202: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:00:31,203: INFO: common: created directory at: artifacts]
[2025-02-15 21:00:31,204: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:02:37,102: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:02:37,104: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:02:37,106: INFO: common: created directory at: artifacts]
[2025-02-15 21:02:37,107: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:03:40,570: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:03:40,573: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:03:40,573: INFO: common: created directory at: artifacts]
[2025-02-15 21:03:40,575: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:04:26,917: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:04:26,919: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:04:26,920: INFO: common: created directory at: artifacts]
[2025-02-15 21:04:26,920: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:04:37,049: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:04:37,053: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:04:37,054: INFO: common: created directory at: artifacts]
[2025-02-15 21:04:37,056: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:06:34,160: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:06:34,163: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:06:34,164: INFO: common: created directory at: artifacts]
[2025-02-15 21:06:34,165: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:06:58,056: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 21:06:58,369: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:06:58,375: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:06:58,375: INFO: common: created directory at: artifacts]
[2025-02-15 21:06:58,377: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:16:48,183: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:16:48,185: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:16:48,185: INFO: common: created directory at: artifacts]
[2025-02-15 21:16:48,185: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:25:13,539: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:25:13,541: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:25:13,541: INFO: common: created directory at: artifacts]
[2025-02-15 21:25:13,541: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:28:57,968: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:28:57,970: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:28:57,971: INFO: common: created directory at: artifacts]
[2025-02-15 21:28:57,971: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:30:29,051: INFO: rouge_scorer: Using default tokenizer.]
[2025-02-15 21:33:10,402: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:33:10,407: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:33:10,409: INFO: common: created directory at: artifacts]
[2025-02-15 21:33:10,410: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 21:34:40,149: INFO: rouge_scorer: Using default tokenizer.]
[2025-02-15 21:52:23,933: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 21:52:24,424: INFO: main: logging is implemented]
[2025-02-15 21:52:24,424: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 21:52:24,434: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:52:24,434: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:52:24,434: INFO: common: created directory at: artifacts]
[2025-02-15 21:52:24,434: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 21:52:24,434: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 21:52:24,533: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 21:52:24,533: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 21:52:24,533: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:52:24,533: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:52:24,533: INFO: common: created directory at: artifacts]
[2025-02-15 21:52:24,533: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 21:52:25,736: ERROR: main: [Errno 22] Invalid argument: 'C:/Users/amitt/Documents/ML-OPS/text-summarization/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 38, in <module>
    data_processing_pipeline.initiate_data_transformation()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_processing_pipeline.py", line 20, in initiate_data_transformation
    data_transformation.convert()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_transformation.py", line 41, in convert
    dataset_samsum_pt.save_to_disk(os.path.join(self.config.root_dir,"samsum_dataset"))
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\datasets\dataset_dict.py", line 1284, in save_to_disk
    dataset.save_to_disk(
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\datasets\arrow_dataset.py", line 1560, in save_to_disk
    for job_id, done, content in Dataset._save_to_disk_single(**kwargs):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\datasets\arrow_dataset.py", line 1584, in _save_to_disk_single
    writer = ArrowWriter(
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\datasets\arrow_writer.py", line 373, in __init__
    self.stream = self._fs.open(path, "wb")
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\fsspec\spec.py", line 1303, in open
    f = self._open(
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\fsspec\implementations\local.py", line 195, in _open
    return LocalFileOpener(path, mode, fs=self, **kwargs)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\fsspec\implementations\local.py", line 359, in __init__
    self._open()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\fsspec\implementations\local.py", line 364, in _open
    self.f = open(self.path, mode=self.mode)
OSError: [Errno 22] Invalid argument: 'C:/Users/amitt/Documents/ML-OPS/text-summarization/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow'
[2025-02-15 21:54:09,319: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 21:54:09,652: INFO: main: logging is implemented]
[2025-02-15 21:54:09,652: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 21:54:09,655: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:54:09,655: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:54:09,655: INFO: common: created directory at: artifacts]
[2025-02-15 21:54:09,655: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 21:54:11,321: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 2737:35D4B8:88AD0D:916C86:67B0D4C2
Accept-Ranges: bytes
Date: Sat, 15 Feb 2025 17:54:11 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990026-FJR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1739642051.547026,VS0,VE897
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 380a10702a7ac47693f1b1c96d0a52100a06a9ed
Expires: Sat, 15 Feb 2025 17:59:11 GMT
Source-Age: 0

]
[2025-02-15 21:54:11,436: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 21:54:11,436: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 21:54:11,436: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:54:11,436: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:54:11,436: INFO: common: created directory at: artifacts]
[2025-02-15 21:54:11,436: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 21:54:13,971: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 21:54:13,971: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 21:54:13,971: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 21:54:13,971: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 21:54:13,973: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 21:54:13,974: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 21:54:13,975: INFO: common: created directory at: artifacts]
[2025-02-15 21:54:13,975: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:00:27,632: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:00:28,090: INFO: main: logging is implemented]
[2025-02-15 22:00:28,090: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:00:28,090: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:00:28,097: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:00:28,097: INFO: common: created directory at: artifacts]
[2025-02-15 22:00:28,097: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:00:28,097: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:00:28,194: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:00:28,194: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:00:28,194: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:00:28,194: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:00:28,201: INFO: common: created directory at: artifacts]
[2025-02-15 22:00:28,201: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:00:29,354: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:00:29,354: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:00:29,354: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:00:29,354: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:00:29,354: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:00:29,354: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:00:29,354: INFO: common: created directory at: artifacts]
[2025-02-15 22:00:29,354: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:05:08,920: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:05:09,331: INFO: main: logging is implemented]
[2025-02-15 22:05:09,331: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:05:09,337: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:05:09,337: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:05:09,337: INFO: common: created directory at: artifacts]
[2025-02-15 22:05:09,337: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:05:09,337: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:05:09,432: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:05:09,432: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:05:09,432: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:05:09,432: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:05:09,432: INFO: common: created directory at: artifacts]
[2025-02-15 22:05:09,432: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:05:10,525: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:05:10,527: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:05:10,527: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:05:10,527: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:05:10,528: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:05:10,529: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:05:10,530: INFO: common: created directory at: artifacts]
[2025-02-15 22:05:10,530: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:06:36,179: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:06:36,630: INFO: main: logging is implemented]
[2025-02-15 22:06:36,630: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:06:36,630: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:06:36,630: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:06:36,630: INFO: common: created directory at: artifacts]
[2025-02-15 22:06:36,630: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:06:36,630: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:06:36,730: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:06:36,730: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:06:36,735: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:06:36,736: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:06:36,736: INFO: common: created directory at: artifacts]
[2025-02-15 22:06:36,736: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:06:37,889: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:06:37,889: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:06:37,889: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:06:37,889: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:06:37,889: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:06:37,889: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:06:37,889: INFO: common: created directory at: artifacts]
[2025-02-15 22:06:37,895: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:06:37,895: ERROR: main: name 'gg' is not defined]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 76, in <module>
    model_evalaution_pipeline.initiate_model_evaluate()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_model_evaluation_pipeline.py", line 20, in initiate_model_evaluate
    model_evaluate.evaluate_model()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 59, in evaluate_model
    gg
NameError: name 'gg' is not defined
[2025-02-15 22:07:16,269: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:07:16,649: INFO: main: logging is implemented]
[2025-02-15 22:07:16,649: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:07:16,653: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:07:16,653: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:07:16,653: INFO: common: created directory at: artifacts]
[2025-02-15 22:07:16,655: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:07:16,656: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:07:16,746: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:07:16,746: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:07:16,746: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:07:16,746: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:07:16,753: INFO: common: created directory at: artifacts]
[2025-02-15 22:07:16,753: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:07:17,796: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:07:17,802: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:07:17,802: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:07:17,802: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:07:17,802: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:07:17,802: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:07:17,802: INFO: common: created directory at: artifacts]
[2025-02-15 22:07:17,802: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:08:08,948: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:08:09,400: INFO: main: logging is implemented]
[2025-02-15 22:08:09,400: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:08:09,400: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:08:09,400: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:08:09,400: INFO: common: created directory at: artifacts]
[2025-02-15 22:08:09,400: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:08:09,408: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:08:09,495: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:08:09,495: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:08:09,495: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:08:09,495: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:08:09,495: INFO: common: created directory at: artifacts]
[2025-02-15 22:08:09,495: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:08:11,005: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:08:11,005: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:08:11,005: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:08:11,005: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:08:11,005: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:08:11,005: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:08:11,005: INFO: common: created directory at: artifacts]
[2025-02-15 22:08:11,005: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:08:48,363: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:08:48,749: INFO: main: logging is implemented]
[2025-02-15 22:08:48,749: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:08:48,749: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:08:48,749: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:08:48,749: INFO: common: created directory at: artifacts]
[2025-02-15 22:08:48,749: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:08:48,749: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:08:48,855: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:08:48,855: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:08:48,855: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:08:48,855: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:08:48,855: INFO: common: created directory at: artifacts]
[2025-02-15 22:08:48,855: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:08:49,898: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:08:49,898: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:08:49,898: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:08:49,898: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:08:49,900: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:08:49,902: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:08:49,902: INFO: common: created directory at: artifacts]
[2025-02-15 22:08:49,902: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:09:20,466: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:09:20,904: INFO: main: logging is implemented]
[2025-02-15 22:09:20,904: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:09:20,904: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:09:20,904: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:09:20,904: INFO: common: created directory at: artifacts]
[2025-02-15 22:09:20,904: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:09:20,904: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:09:21,005: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:09:21,008: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:09:21,008: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:09:21,008: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:09:21,008: INFO: common: created directory at: artifacts]
[2025-02-15 22:09:21,008: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:09:22,056: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:09:22,056: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:09:22,056: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:09:22,056: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:09:22,056: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:09:22,056: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:09:22,056: INFO: common: created directory at: artifacts]
[2025-02-15 22:09:22,056: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:09:22,056: ERROR: main: name 'hh' is not defined]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 76, in <module>
    model_evalaution_pipeline.initiate_model_evaluate()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_model_evaluation_pipeline.py", line 20, in initiate_model_evaluate
    model_evaluate.evaluate_model()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 64, in evaluate_model
    hh
NameError: name 'hh' is not defined
[2025-02-15 22:09:56,569: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:09:56,909: INFO: main: logging is implemented]
[2025-02-15 22:09:56,909: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:09:56,909: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:09:56,909: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:09:56,909: INFO: common: created directory at: artifacts]
[2025-02-15 22:09:56,909: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:09:56,909: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:09:57,007: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:09:57,007: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:09:57,007: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:09:57,007: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:09:57,007: INFO: common: created directory at: artifacts]
[2025-02-15 22:09:57,007: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:09:58,030: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:09:58,030: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:09:58,030: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:09:58,030: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:09:58,032: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:09:58,033: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:09:58,035: INFO: common: created directory at: artifacts]
[2025-02-15 22:09:58,035: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:35:11,164: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:35:11,585: INFO: main: logging is implemented]
[2025-02-15 22:35:11,585: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:11,595: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:11,595: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:11,595: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:11,595: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:35:11,595: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:35:11,682: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:35:11,682: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:35:11,682: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:11,697: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:11,697: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:11,697: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:35:12,932: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:35:12,932: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:12,932: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:35:12,932: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:12,932: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:12,935: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:12,935: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:12,935: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:35:17,010: ERROR: main: name 'nn' is not defined]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 76, in <module>
    model_evalaution_pipeline.initiate_model_evaluate()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_model_evaluation_pipeline.py", line 20, in initiate_model_evaluate
    model_evaluate.evaluate_model()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 71, in evaluate_model
    nn
NameError: name 'nn' is not defined
[2025-02-15 22:35:33,667: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:35:34,009: INFO: main: logging is implemented]
[2025-02-15 22:35:34,009: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:34,009: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:34,009: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:34,009: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:34,009: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:35:34,009: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:35:34,113: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:35:34,113: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:35:34,113: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:34,113: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:34,113: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:34,113: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:35:35,145: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:35:35,145: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:35,145: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:35:35,145: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:35,145: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:35,145: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:35,145: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:35,145: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:35:38,749: ERROR: main: name 'ww' is not defined]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 76, in <module>
    model_evalaution_pipeline.initiate_model_evaluate()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_model_evaluation_pipeline.py", line 20, in initiate_model_evaluate
    model_evaluate.evaluate_model()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 74, in evaluate_model
    ww
NameError: name 'ww' is not defined
[2025-02-15 22:35:55,277: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:35:55,626: INFO: main: logging is implemented]
[2025-02-15 22:35:55,626: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:55,630: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:55,630: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:55,630: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:55,630: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:35:55,630: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:35:55,723: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:35:55,723: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:35:55,723: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:55,726: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:55,726: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:55,726: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:35:56,712: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:35:56,712: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:56,712: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:35:56,713: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:35:56,714: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:35:56,715: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:35:56,716: INFO: common: created directory at: artifacts]
[2025-02-15 22:35:56,716: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:36:00,550: ERROR: main: name 'll' is not defined]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 76, in <module>
    model_evalaution_pipeline.initiate_model_evaluate()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_model_evaluation_pipeline.py", line 20, in initiate_model_evaluate
    model_evaluate.evaluate_model()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 79, in evaluate_model
    ll
NameError: name 'll' is not defined
[2025-02-15 22:36:41,063: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:36:41,431: INFO: main: logging is implemented]
[2025-02-15 22:36:41,431: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:36:41,431: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:36:41,431: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:36:41,431: INFO: common: created directory at: artifacts]
[2025-02-15 22:36:41,431: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:36:41,431: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:36:41,528: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:36:41,528: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:36:41,528: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:36:41,535: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:36:41,535: INFO: common: created directory at: artifacts]
[2025-02-15 22:36:41,535: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:36:42,577: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:36:42,577: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:36:42,577: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:36:42,577: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:36:42,577: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:36:42,577: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:36:42,577: INFO: common: created directory at: artifacts]
[2025-02-15 22:36:42,583: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:36:46,576: ERROR: main: name 'mm' is not defined]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 76, in <module>
    model_evalaution_pipeline.initiate_model_evaluate()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_model_evaluation_pipeline.py", line 20, in initiate_model_evaluate
    model_evaluate.evaluate_model()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 81, in evaluate_model
    score = self.calculate_metric_on_test_ds(dataset_samsum_pt['test'][0:10], rouge_metric, model, tokenizer, batch_size = 2, column_text = 'dialogue', column_summary= 'summary')
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 33, in calculate_metric_on_test_ds
    mm
NameError: name 'mm' is not defined
[2025-02-15 22:37:40,597: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:37:40,972: INFO: main: logging is implemented]
[2025-02-15 22:37:40,972: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:37:40,978: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:37:40,979: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:37:40,979: INFO: common: created directory at: artifacts]
[2025-02-15 22:37:40,979: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:37:40,979: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:37:41,077: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:37:41,077: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:37:41,077: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:37:41,077: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:37:41,077: INFO: common: created directory at: artifacts]
[2025-02-15 22:37:41,077: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:37:42,125: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:37:42,132: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:37:42,132: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:37:42,132: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:37:42,133: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:37:42,133: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:37:42,133: INFO: common: created directory at: artifacts]
[2025-02-15 22:37:42,133: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:37:46,227: ERROR: main: name 'hh' is not defined]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 76, in <module>
    model_evalaution_pipeline.initiate_model_evaluate()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_model_evaluation_pipeline.py", line 20, in initiate_model_evaluate
    model_evaluate.evaluate_model()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 84, in evaluate_model
    score = self.calculate_metric_on_test_ds(dataset_samsum_pt['test'][0:10], rouge_metric, model, tokenizer, batch_size = 2, column_text = 'dialogue', column_summary= 'summary')
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 35, in calculate_metric_on_test_ds
    article_batches = list(self.generate_batch_sized_chunks(dataset[column_text], batch_size))
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 22, in generate_batch_sized_chunks
    hh
NameError: name 'hh' is not defined
[2025-02-15 22:38:16,254: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:38:16,609: INFO: main: logging is implemented]
[2025-02-15 22:38:16,609: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:38:16,618: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:38:16,618: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:38:16,618: INFO: common: created directory at: artifacts]
[2025-02-15 22:38:16,618: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:38:16,618: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:38:16,712: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:38:16,712: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:38:16,712: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:38:16,712: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:38:16,712: INFO: common: created directory at: artifacts]
[2025-02-15 22:38:16,712: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:38:17,740: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:38:17,740: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:38:17,740: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:38:17,740: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:38:17,740: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:38:17,740: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:38:17,740: INFO: common: created directory at: artifacts]
[2025-02-15 22:38:17,740: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:38:21,652: ERROR: main: name 'rr' is not defined]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 76, in <module>
    model_evalaution_pipeline.initiate_model_evaluate()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_model_evaluation_pipeline.py", line 20, in initiate_model_evaluate
    model_evaluate.evaluate_model()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 84, in evaluate_model
    score = self.calculate_metric_on_test_ds(dataset_samsum_pt['test'][0:10], rouge_metric, model, tokenizer, batch_size = 2, column_text = 'dialogue', column_summary= 'summary')
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\model_evaluation.py", line 38, in calculate_metric_on_test_ds
    rr
NameError: name 'rr' is not defined
[2025-02-15 22:38:38,771: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:38:39,108: INFO: main: logging is implemented]
[2025-02-15 22:38:39,108: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:38:39,108: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:38:39,108: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:38:39,108: INFO: common: created directory at: artifacts]
[2025-02-15 22:38:39,108: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:38:39,108: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:38:39,216: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:38:39,216: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:38:39,216: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:38:39,216: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:38:39,216: INFO: common: created directory at: artifacts]
[2025-02-15 22:38:39,216: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:38:40,304: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:38:40,304: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:38:40,304: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:38:40,304: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:38:40,307: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:38:40,308: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:38:40,309: INFO: common: created directory at: artifacts]
[2025-02-15 22:38:40,309: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:39:47,646: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 22:39:47,984: INFO: main: logging is implemented]
[2025-02-15 22:39:47,984: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:39:47,993: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:39:47,993: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:39:47,993: INFO: common: created directory at: artifacts]
[2025-02-15 22:39:47,993: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 22:39:47,993: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 22:39:48,091: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:39:48,091: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 22:39:48,091: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:39:48,091: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:39:48,091: INFO: common: created directory at: artifacts]
[2025-02-15 22:39:48,091: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 22:39:49,149: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 22:39:49,149: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:39:49,149: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 22:39:49,149: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 22:39:49,149: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 22:39:49,149: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 22:39:49,149: INFO: common: created directory at: artifacts]
[2025-02-15 22:39:49,149: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 22:41:12,585: INFO: rouge_scorer: Using default tokenizer.]
[2025-02-15 22:41:12,672: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 23:18:52,208: INFO: server: Started server process [26376]]
[2025-02-15 23:18:52,208: INFO: on: Waiting for application startup.]
[2025-02-15 23:18:52,208: INFO: on: Application startup complete.]
[2025-02-15 23:18:52,208: INFO: server: Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)]
[2025-02-15 23:21:36,704: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 23:21:36,704: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 23:21:36,704: INFO: common: created directory at: artifacts]
[2025-02-15 23:21:36,704: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 23:22:24,496: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 23:22:24,499: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 23:22:24,499: INFO: common: created directory at: artifacts]
[2025-02-15 23:22:24,499: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 23:24:06,878: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 23:24:06,879: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 23:24:06,880: INFO: common: created directory at: artifacts]
[2025-02-15 23:24:06,881: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-15 23:27:01,937: INFO: server: Shutting down]
[2025-02-15 23:27:02,049: INFO: on: Waiting for application shutdown.]
[2025-02-15 23:27:02,049: INFO: on: Application shutdown complete.]
[2025-02-15 23:27:02,049: INFO: server: Finished server process [26376]]
[2025-02-16 22:47:29,444: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-16 22:47:29,444: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-16 22:47:33,431: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:47:33] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 22:47:33,470: ERROR: app: Exception on /docs [GET]]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\app_new.py", line 16, in docs
    return render_template("docs.html")
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: docs.html
[2025-02-16 22:47:33,485: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:47:33] "[35m[1mGET /docs HTTP/1.1[0m" 500 -]
[2025-02-16 22:47:33,535: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:47:33] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-02-16 22:49:15,322: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-16 22:49:15,322: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-16 22:49:17,708: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:49:17] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 22:49:17,774: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:49:17] "GET /docs HTTP/1.1" 200 -]
[2025-02-16 22:49:32,619: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:49:32] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 22:49:32,643: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:49:32] "GET /docs HTTP/1.1" 200 -]
[2025-02-16 22:50:39,140: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:50:39] "[31m[1mGET /predict HTTP/1.1[0m" 405 -]
[2025-02-16 22:51:07,180: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-16 22:51:08,482: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:51:08] "[31m[1mGET /predict HTTP/1.1[0m" 405 -]
[2025-02-16 22:51:08,667: INFO: main: logging is implemented]
[2025-02-16 22:51:08,667: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-16 22:51:08,677: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:51:08,687: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:51:08,688: INFO: common: created directory at: artifacts]
[2025-02-16 22:51:08,688: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-16 22:51:08,689: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-16 22:51:08,797: INFO: main: stage { STAGE_NAME } completed]
[2025-02-16 22:51:08,797: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-16 22:51:08,805: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:51:08,805: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:51:08,805: INFO: common: created directory at: artifacts]
[2025-02-16 22:51:08,805: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-16 22:51:10,753: INFO: main: stage {STAGE_NAME} completed]
[2025-02-16 22:51:10,753: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-16 22:51:10,754: INFO: main: stage { STAGE_NAME } completed]
[2025-02-16 22:51:10,755: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-16 22:51:10,759: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:51:10,759: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:51:10,759: INFO: common: created directory at: artifacts]
[2025-02-16 22:51:10,759: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-16 22:54:34,352: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-16 22:54:34,352: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-16 22:54:37,317: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:54:37] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 22:54:37,361: ERROR: app: Exception on /predict [GET]]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\app_new.py", line 43, in predict_route
    return render_template("predict.html", summary=summary)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: predict.html
[2025-02-16 22:54:37,376: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:54:37] "[35m[1mGET /predict HTTP/1.1[0m" 500 -]
[2025-02-16 22:54:42,875: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:54:42] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 22:54:42,881: ERROR: app: Exception on /predict [GET]]
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\app_new.py", line 43, in predict_route
    return render_template("predict.html", summary=summary)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\myenv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: predict.html
[2025-02-16 22:54:42,882: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:54:42] "[35m[1mGET /predict HTTP/1.1[0m" 500 -]
[2025-02-16 22:55:29,159: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-16 22:55:29,159: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-16 22:55:31,973: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:55:31] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 22:55:32,020: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:55:32] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 22:55:46,086: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:55:46,090: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:55:46,092: INFO: common: created directory at: artifacts]
[2025-02-16 22:55:46,094: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-16 22:56:00,792: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:56:00] "POST /predict HTTP/1.1" 200 -]
[2025-02-16 22:57:39,537: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:57:39,538: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:57:39,539: INFO: common: created directory at: artifacts]
[2025-02-16 22:57:39,539: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-16 22:57:53,356: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:57:53] "POST /predict HTTP/1.1" 200 -]
[2025-02-16 22:59:31,517: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-16 22:59:31,517: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-16 22:59:33,182: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:59:33] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 22:59:33,252: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:59:33] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 22:59:42,933: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:59:42] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 22:59:44,004: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:59:44] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 22:59:44,366: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:59:44] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 22:59:44,583: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:59:44] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 22:59:54,184: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-16 22:59:54,184: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-16 22:59:56,697: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:59:56] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 22:59:56,753: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 22:59:56] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 23:01:29,841: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 23:01:29] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 23:04:37,278: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-16 23:04:37,278: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-16 23:04:41,608: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 23:04:41] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 23:04:41,677: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 23:04:41] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 23:04:43,631: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 23:04:43] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-16 23:04:43,660: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 23:04:43] "GET /predict HTTP/1.1" 200 -]
[2025-02-16 23:05:15,491: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 23:05:15,504: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 23:05:15,505: INFO: common: created directory at: artifacts]
[2025-02-16 23:05:15,508: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-16 23:06:01,369: INFO: _internal: 127.0.0.1 - - [16/Feb/2025 23:06:01] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 10:46:51,089: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 10:46:51,104: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 10:46:51,110: INFO: common: created directory at: artifacts]
[2025-02-17 10:46:51,111: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 10:47:42,307: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 10:47:42,308: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 10:47:46,141: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 10:47:46] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 10:47:46,253: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 10:47:46] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 10:47:56,242: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 10:47:56] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 10:47:56,919: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 10:47:56] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 10:48:05,122: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 10:48:05,122: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 10:48:08,092: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 10:48:08] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 10:48:08,134: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 10:48:08] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 10:55:17,330: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 10:55:17,330: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 10:55:19,827: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 10:55:19] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 10:55:19,908: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 10:55:19] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 11:46:05,186: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 11:46:05,186: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 11:46:14,356: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 11:46:14] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 11:46:14,426: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 11:46:14] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 11:46:16,703: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 11:46:16] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 11:46:16,743: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 11:46:16] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:05:18,056: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:05:18,056: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:05:22,177: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:05:22] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:05:22,232: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:05:22] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:06:39,690: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:06:39,690: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:06:55,036: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:06:55] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:06:55,109: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:06:55] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:07:44,837: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:07:44,837: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:07:47,114: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:07:47] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:07:47,194: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:07:47] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:18:13,970: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:18:13,970: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:18:16,643: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:18:16] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:18:16,726: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:18:16] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:19:29,195: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:19:29,196: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:19:33,343: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:19:33] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:19:33,408: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:19:33] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:20:32,552: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:20:32,552: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:20:36,023: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:20:36] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:20:36,086: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:20:36] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:20:56,188: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:20:56,188: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:20:59,458: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:20:59] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:20:59,503: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:20:59] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:21:28,687: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:21:28,687: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:21:31,464: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:21:31] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:21:31,491: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:21:31] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:21:58,373: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 12:21:58,373: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 12:22:00,298: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:22:00] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:22:01,561: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:22:01] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:22:01,586: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:22:01] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:22:03,495: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:22:03] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 12:22:03,520: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:22:03] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 12:22:26,004: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 12:22:26,007: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 12:22:26,009: INFO: common: created directory at: artifacts]
[2025-02-17 12:22:26,011: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 12:23:12,467: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 12:23:12] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:05:35,033: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 13:05:35,033: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 13:05:51,451: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:05:51] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:05:51,514: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:05:51] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:05:53,341: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:05:53] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:05:53,374: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:05:53] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:06:21,387: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 13:06:21,387: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 13:06:26,916: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:06:26] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:06:26,975: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:06:26] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:07:00,393: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 13:07:00,393: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 13:07:02,431: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:07:02] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:07:02,502: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:07:02] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:08:34,778: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 13:08:34,778: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 13:08:37,199: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:08:37] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:08:37,252: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:08:37] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:08:39,114: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:08:39] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:08:39,140: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:08:39] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:09:45,663: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 13:09:45,663: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 13:09:48,682: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:09:48] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:09:48,759: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:09:48] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:11:29,159: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:11:29,160: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:11:29,161: INFO: common: created directory at: artifacts]
[2025-02-17 13:11:29,163: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:11:56,254: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:11:56,259: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:11:56,269: INFO: common: created directory at: artifacts]
[2025-02-17 13:11:56,272: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:11:58,670: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:11:58,722: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:11:58,756: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:11:58,784: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:11:58,835: INFO: common: created directory at: artifacts]
[2025-02-17 13:11:58,937: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:11:58,937: INFO: common: created directory at: artifacts]
[2025-02-17 13:11:58,956: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:11:59,066: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:11:59,143: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:11:59,259: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:11:59,263: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:11:59,270: INFO: common: created directory at: artifacts]
[2025-02-17 13:11:59,431: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:11:59,431: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:11:59,556: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:11:59,556: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:11:59,508: INFO: common: created directory at: artifacts]
[2025-02-17 13:12:00,065: INFO: common: created directory at: artifacts]
[2025-02-17 13:12:00,471: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:12:00,482: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:12:00,483: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:12:00,581: INFO: common: created directory at: artifacts]
[2025-02-17 13:12:00,608: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:12:49,956: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:12:49] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:13:49,122: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:13:49] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:15:20,093: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:15:20] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:15:22,261: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:15:22] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:15:22,277: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:15:22] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:15:24,558: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:15:24] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:15:25,211: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:15:25] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:15:26,398: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:15:26] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:19:38,772: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 13:19:38,772: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 13:19:43,373: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:19:43] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:19:43,433: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:19:43] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:20:24,662: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:20:24,669: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:20:24,670: INFO: common: created directory at: artifacts]
[2025-02-17 13:20:24,670: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:21:29,314: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:21:29] "POST /predict HTTP/1.1" 200 -]
[2025-02-17 13:25:47,866: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.15:8080]
[2025-02-17 13:25:47,866: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-02-17 13:25:56,850: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:25:56] "[32mGET / HTTP/1.1[0m" 302 -]
[2025-02-17 13:25:56,893: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:25:56] "GET /predict HTTP/1.1" 200 -]
[2025-02-17 13:26:01,321: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 13:26:01,322: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 13:26:01,323: INFO: common: created directory at: artifacts]
[2025-02-17 13:26:01,323: INFO: common: created directory at: artifacts/model_evaluation]
[2025-02-17 13:28:11,747: INFO: _internal: 127.0.0.1 - - [17/Feb/2025 13:28:11] "POST /predict HTTP/1.1" 200 -]
